features/steps/cocoa_steps.rb

```diff
@@ -3,57 +3,158 @@
 end
 
 When("I run {string} and relaunch the crashed app") do |event_type|
-  begin
-    step("I run \"#{event_type}\"")
-  rescue StandardError
-    # Ignore on macOS - AppiumForMac raises an error when clicking a button causes the app to crash
-    raise unless Maze::Helper.get_current_platform.eql?('macos')
```

Comment:
Since this capture is only here to ignore these on MacOS we can removed it and allow the error to raise naturally.